srcalc/src/main/webapp/css/srcalc.css

```diff
@@ -95,20 +95,24 @@ footer {
 /* Keep a standard height for all buttons on the application.
 */
 button {
-	height: 25px;
-	border: 1px solid #888;
+    height: 25px;
```

Comment:
This height does not apply to a.btn-link elements. We should make it apply somehow.